.github/workflows/api-integration-tests.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           coverage: none
 
       - name: Setup BATS & httpie
-        run: sudo apt-get install -y bats httpie
+        run: sudo apt-get install -y httpie && npm install -g bats@1.7.0
```

Comment:
Wait, why use npm for this?

Reply:
Because the Ubuntu package has like 1.1 in their repo slightly newer on new versions.
There is also no backports. For the basic stuff it  will work but of course there are details that are different.

I can't work with that though latest is 1.8 but I didn't test that yet on my machine since it didn't land in manjaro as of now.

npm is one of the projects official distribution ways (not sure why), I checked the repo for alternative install methods after I noticed that Ubuntu is hanging behind.

And since npm is available in the action image I think this is the best way.

Reply:
I will probably in the future also add a requires bats version X line to the bats files.

I have no time to debug issues because of bugs that were already fixed by the project.

Reply:
Let's start like this then